cartridges/int_adyen_SFRA/cartridge/controllers/CheckoutServices.js

```diff
@@ -30,7 +31,7 @@ server.prepend('PlaceOrder', server.middleware.https, function (req, res, next)
     }
 
     collections.forEach(currentBasket.getPaymentInstruments(), function (paymentInstrument) {
-        if(paymentInstrument.paymentMethod == "Adyen" || paymentInstrument.paymentMethod == paymentInstrument.METHOD_CREDIT_CARD){
+        if([constants.METHOD_ADYEN, paymentInstrument.METHOD_CREDIT_CARD, constants.METHOD_ADYEN_POS].indexOf(paymentInstrument.paymentMethod) !== -1){
```

Comment:
This condition is extremely hard to search for. If you're not the one providing maintenance to this part, the person in charge will have a hard time.